examples/aws-cloudwatch/capacity_planning/rds.juttle

```diff
@@ -47,8 +46,7 @@ import "examples/aws-cloudwatch/common/capacity_planning_cpu.juttle" as capacity
  | split
  | view piechart -title "AWS Capacity Planning (Iops) for RDS";
 
-(read cloudwatch -period 3600 -last :1h: product='RDS'
- | filter name='FreeStorageSpace'
+(read cloudwatch -period 3600 -last :1h: metric="RDS:FreeStorageSpace"
```

Comment:
Outside of this PR we should think about whether we want to stick with always using durations in adapter options instead of number of seconds (i.e. :1h: instead 3600).

I believe we used to have `batch` and `reduce -every` support both but now they only support durations.